modules/storage-azure-file-rbac.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage/persistent-storage-azure-file.adoc
+
+[id="configuring-rbac-for-azure-file_{context}"]
+= Configuring RBAC for the Azure File StorageClass
+
+The StorageClass uses secrets to store the storage account name
+and the storage account key required to create an Azure Files share.
```

Comment:
s/required/that are required

Please confirm that "Azure Files share" is right. You use "Azure File share" later, and that scans better.